src/test/java/com/radixdlt/serialization/SerializationTestUtilsEngine.java

```diff
@@ -0,0 +1,55 @@
+package com.radixdlt.serialization;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.flipkart.zjsonpatch.JsonDiff;
+
+import java.io.IOException;
+
+import static org.junit.Assert.fail;
+
+/**
+ * Utilities for test encoding and decoding with serializers.
+ */
+final class SerializationTestUtilsEngine {
```

Comment:
Uh, please, fix this! Copy paste of `SerializationTestUtils` basically.... bah!

Reply:
Yeap.  There is still some duplicate stuff hanging around from where the engine was split from core.